src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$star-active: url("/images/star-active.svg");
+$star: url("/images/star.svg");
+$bg-image: url(../images/imac.jpeg);
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  width: 198px;
```

Comment:
use `box-sizing: border-box;` property for all elements on the page (`*` selector), and set width including border - 200px here